waiter/integration/waiter/saml_authentication_test.clj

```diff
@@ -0,0 +1,111 @@
+(ns waiter.saml-authentication-test
+  (:require [clojure.data.json :as json]
+            [clojure.string :as string]
+            [clojure.test :refer :all]
+            [waiter.util.client-tools :refer :all]
+            [clojure.java.shell :as shell])
+  (:import (java.net URLEncoder)))
+
+(deftest ^:parallel ^:integration-fast test-default-composite-authenticator
+  (testing-using-waiter-url
+    (when (using-shell? waiter-url)
+      (let [run-as-user (System/getenv "WAITER_AUTH_RUN_AS_USER")
+            _ (is (not (string/blank? run-as-user)) "You must provide the :one-user authenticator login in the WAITER_AUTH_RUN_AS_USER environment variable")
```

Comment:
Nitpick: move the assertion message to a new line

Reply:
done